_posts/en/newsletters/2021-10-06-newsletter.md

```diff
@@ -0,0 +1,167 @@
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core GUI #416][] adds an "Enable RPC server" checkbox which allows
+  the user to turn on and off Bitcoin Core's RPC server (requires restart).
+
+    {:.center}
+    ![Screenshot of the Enable RPC server configuration option](/img/posts/2021-10-gui-rpc-server.png)
+
+- [Bitcoin Core #20591][] changes the wallet time calculation logic to
+  only use the block timestamp when rescanning historical blocks for
+transactions relevant to the wallet. Users and applications using the
+`rescanblockchain` RPC to manually invoke rescans should no longer see
+transactions inaccurately labeled with the time scanned instead of
+time confirmed, a bug diagnosed and fixed by a patient downstream
+developer looking after the Bitcoin ecosystem.
+
+- [Bitcoin Core #22722][] updates the `estimatesmartfee` RPC so that it
+  only returns feerates higher than both the configured and dynamic
+  minimum transaction relay fees.  For example, if the estimator
+  calculates a fee of 1 sat/vbyte, the configured value is 2 sat/vbyte,
+  and the dynamic minimum has risen to 3 sat/vbyte, then a value of 3
+  sat/vbyte will be returned.
+
+- [Bitcoin Core #17526][] adds the Single Random Draw (SRD) algorithm as
+  a third coin selection strategy. The wallet will now acquire coin
+  selection results from each the Branch and Bound (BnB), the knapsack,
+  and the SRD algorithm and use the [waste heuristic][news165 waste] we
+  described previously to select the most cost effective coin selection
+  result out of the three to fund the transaction.
+
+  In simulations based on about 8,000 payments, the PR author
+  found that the addition of the SRD algorithm reduced overall
+  transaction fees by 6% and increased the occurrence of changeless
+  transactions from 5.4% to 9.0%. Not creating a change output decreases
+  the weight and fees of a transaction, reduces the wallet's UTXO pool
+  size, saves the cost of later spending the change output, and is
+  thought to improve the privacy of the wallet.
+
+- [Bitcoin Core #23061][] fixes the `-persistmempool` configuration
+  option, which previously did not persist the mempool to disk on
+  shutdown when no parameter was passed (to actually persist, you had to
+  pass `-persistmempool=1`).  Now using the bare option will persist the
+  mempool.
+
+- [Bitcoin Core #23065][] makes it possible for wallet UTXO locks to be
+  persisted to disk.  Bitcoin Core allows users to lock one or more of their
+  UTXOs to prevent it from being used in automatically-created
+  transactions.  The `lockunspent` RPC now includes a `persistent`
+  parameter that saves the preference to disk, and the GUI automatically
+  persists user-selected locks to disk.  One use for persistent locks is
+  preventing spending of [dust spam][topic output linking] outputs, or
```

Comment:
I have a slight preference to not use the word 'dust' to describe this kind of privacy attack, since it's a different meaning from the p2p relay policy concept of dust.